cmd/solarwinds-otel-collector/main.go

```diff
@@ -65,6 +65,7 @@ func main() {
 
 func run(params otelcol.CollectorSettings) error {
 	cmd := otelcol.NewCommand(params)
+	addCommand(cmd)
```

Comment:
```suggestion
	addCommands(cmd)
```
Nit: We could prepare for new commands in the future.

Reply:
fixed